app/models/form/lettings/questions/nationality_all.rb

```diff
@@ -0,0 +1,222 @@
+    "458" => { "name" => "Malaysia", "synonyms" => "MY" },
+    "462" => { "name" => "Maldives", "synonyms" => "MV" },
+    "466" => { "name" => "Mali", "synonyms" => "ML" },
+    "470" => { "name" => "Malta", "synonyms" => "MT" },
+    "584" => { "name" => "Marshall Islands", "synonyms" => "MH" },
+    "478" => { "name" => "Mauritania", "synonyms" => "MR" },
+    "480" => { "name" => "Mauritius", "synonyms" => "MU" },
+    "484" => { "name" => "Mexico", "synonyms" => "MX" },
+    "583" => { "name" => "Micronesia (Federated States of)", "synonyms" => "" },
+    "498" => { "name" => "Moldova", "synonyms" => "MD" },
+    "492" => { "name" => "Monaco", "synonyms" => "MC" },
+    "496" => { "name" => "Mongolia", "synonyms" => "MN" },
+    "499" => { "name" => "Montenegro", "synonyms" => "ME" },
+    "504" => { "name" => "Morocco", "synonyms" => "MA" },
+    "508" => { "name" => "Mozambique", "synonyms" => "MZ" },
+    "104" => { "name" => "Myanmar", "synonyms" => "" },
+    "516" => { "name" => "Namibia", "synonyms" => "NA" },
+    "520" => { "name" => "Nauru", "synonyms" => "NR" },
+    "524" => { "name" => "Nepal", "synonyms" => "NP" },
+    "528" => { "name" => "Netherlands", "synonyms" => "NL" },
+    "554" => { "name" => "New Zealand", "synonyms" => "NZ" },
+    "558" => { "name" => "Nicaragua", "synonyms" => "NI" },
+    "562" => { "name" => "Niger", "synonyms" => "NE" },
+    "566" => { "name" => "Nigeria", "synonyms" => "NG" },
+    "807" => { "name" => "North Macedonia", "synonyms" => "MK" },
+    "408" => { "name" => "North Korea", "synonyms" => "KP" },
+    "578" => { "name" => "Norway", "synonyms" => "NO" },
+    "512" => { "name" => "Oman", "synonyms" => "OM" },
+    "586" => { "name" => "Pakistan", "synonyms" => "PK" },
+    "585" => { "name" => "Palau", "synonyms" => "PW" },
+    "275" => { "name" => "Palestine, State of", "synonyms" => "" },
+    "591" => { "name" => "Panama", "synonyms" => "PA" },
+    "598" => { "name" => "Papua New Guinea", "synonyms" => "PG" },
+    "600" => { "name" => "Paraguay", "synonyms" => "PY" },
+    "604" => { "name" => "Peru", "synonyms" => "PE" },
+    "608" => { "name" => "Philippines", "synonyms" => "PH" },
+    "616" => { "name" => "Poland", "synonyms" => "PL" },
+    "620" => { "name" => "Portugal", "synonyms" => "PT" },
+    "634" => { "name" => "Qatar", "synonyms" => "QA" },
+    "642" => { "name" => "Romania", "synonyms" => "RO" },
+    "643" => { "name" => "Russia", "synonyms" => "RU" },
+    "646" => { "name" => "Rwanda", "synonyms" => "RW" },
+    "659" => { "name" => "Saint Kitts and Nevis", "synonyms" => "" },
+    "662" => { "name" => "Saint Lucia", "synonyms" => "" },
+    "670" => { "name" => "Saint Vincent and the Grenadines", "synonyms" => "" },
+    "882" => { "name" => "Samoa", "synonyms" => "WS" },
+    "674" => { "name" => "San Marino", "synonyms" => "SM" },
+    "678" => { "name" => "Sao Tome and Principe", "synonyms" => "ST" },
```

Comment:
we should have accents on this (I saw we had on cote d'ivoire) so I don't think we're being consistent accent-wise